coc/events.pyi

```diff
@@ -150,9 +178,9 @@ class ClientEvents:
     @classmethod
     def raid_weekend_end(cls): ...
     @classmethod
-    def clan_games_start(cls): ...
+    def clangames_start(cls): ...
```

Comment:
clan_games_start

Reply:
Yea can change this was from 2.3.0 then I guess